.github/workflows/build_wheels.yml

```diff
@@ -199,16 +199,22 @@ jobs:
         run: gsutil cp -r -a public-read source/* ${{ env.GCP_PATH }}
 
   build_wheels:
-    name: Build python wheels on ${{ matrix.os_python.os }}
+    name: Build python wheels on ${{matrix.arch}} for ${{ matrix.os_python.os }}
     needs: build_source
+    env:
+      CIBW_ARCHS_LINUX: ${{matrix.arch}}
     runs-on: ${{ matrix.os_python.os }}
     strategy:
       matrix:
         os_python: [
-          {"os": "ubuntu-latest", "python": "cp36-* cp37-* cp38-*"},
-          {"os": "macos-latest", "python": "cp36-* cp37-* cp38-*"},
-          {"os": "windows-latest", "python": "cp36-* cp37-* cp38-*"},
+          {"os": "ubuntu-latest", "python": "cp36-* cp37-* cp38-* cp39-*"},
```

Comment:
We don't have test suites for Py39 yet, however this may give impression that we already support it; it would add py39 wheels when we add Py3.9 test suites and officially support it.

Reply:
Ok, I have removed py39.